packages/ricos-content/web/src/converters/draft/fromDraft/convertRicosPluginData.ts

```
@@ -236,15 +236,22 @@ const convertLink = ({
   anchor?: string;
 }): Link => {
   const relValues =
-    rel
-      ?.split(' ')
+    (rel || '')
+      .split(' ')
       .filter(key => ['nofollow', 'sponsored', 'ugc'].includes(key))
       .map(key => [key, true]) || [];
+  let _target;
+  try {
+    _target = target?.toUpperCase().substring(1);
+  } catch (e) {
+    console.warn('Target is not string', target);
```

do we want to hide the error?

---

@oded-soffrin marked as resolved. But I didn't see a comment?

---

I'm written in another comment - i changed the fixture with updated button data, and removed the safety code (not needed if it's not legacy content)